crates/gourgeist/src/main.rs

```diff
@@ -34,7 +34,7 @@ fn run() -> Result<(), gourgeist::Error> {
         Cache::from_path(".gourgeist_cache")?
     };
     let info = Interpreter::query(python.as_std_path(), &platform, &cache).unwrap();
-    create_bare_venv(&location, &info)?;
+    create_bare_venv(&location, &info, None)?;
```

Comment:
This isn't strictly required in this PR but I think it would be useful to provide the flag in the `gourgeist` CLI interface as well. You'll find the argument struct in this file itself which can then be passed here.